<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Visual Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EndoscopeRes.qrc">
    <normaloff>:/icon.jpg</normaloff>:/icon.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="btnStartOrPause">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>248</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>104</y>
     <width>500</width>
     <height>375</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>1050</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcessed">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>104</y>
     <width>500</width>
     <height>375</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1400</width>
     <height>1050</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseApp">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>404</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>180</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>484</y>
     <width>217</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Live endoscopic Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>484</y>
     <width>173</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image Processing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openFrmGrab">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>100</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Open FrameGrabber</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeFrmGrab">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>144</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close FrameGrabber</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>28</y>
     <width>797</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image- and Robotic-based Steering of a Flexible Endoscope in ENT Diagnoses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveImage">
   <property name="geometry">
    <rect>
     <x>1116</x>
     <y>296</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>save Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="TUMLabel">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>24</y>
     <width>85</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MIMEDLabel">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>24</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>btnStartOrPause</zorder>
  <zorder>lblOriginal</zorder>
  <zorder>lblProcessed</zorder>
  <zorder>CloseApp</zorder>
  <zorder>openFrmGrab</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>closeFrmGrab</zorder>
  <zorder>label_3</zorder>
  <zorder>btnSaveImage</zorder>
  <zorder>TUMLabel</zorder>
  <zorder>MIMEDLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EndoscopeRes.qrc"/>
 </resources>
 <connections/>
</ui>
